test/e2e/network/netpol/network_policy.go

```diff
@@ -1059,22 +1059,46 @@ var _ = common.SIGDescribe("Netpol", func() {
 			ValidateOrFail(k8s, model, &TestCase{ToPort: 80, Protocol: v1.ProtocolTCP, Reachability: denyIngressToXReachability})
 		})
 
-		ginkgo.It("should not allow access by TCP when a policy specifies only SCTP [Feature:NetworkPolicy] [Feature:SCTP]", func() {
```

Comment:
We still have the old ones in place, should they be updated accordingly as well?
 https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/netpol/network_legacy.go#L1685

Reply:
ah, fixed